### PR TITLE
[Calls] Calls TURN content updates with analytics

### PR DESCRIPTION
**Pull request by renandincer**

### CODE REVIEW

File: content/calls/turn/what-is-turn.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+3. **TURN Client**: An application or device that uses the TURN protocol to communicate through a TURN server. This is your application. It can be a web applicaition using the WebRTC APIs or a native application running on mobile or desktop.
+
+4. **Allocation**: When a TURN server creates an allocation, the TURN server reserves an IP and a port unique to that client.
+
+5. **Relayed Transport Address**: The IP address and port reserved on the TURN server that others on the Internet can use to send data to the TURN client.
+
+## How TURN Works
+
+1. A TURN client sends an Allocate request to a TURN server.
```
Comment (nils-ohlmeier) [inline content/calls/turn/what-is-turn.md:26]: Do you want to add authentication here?
I think it's totally optional. Just trying to make sure it's not forgotten if deemed important.

File: content/calls/turn/replacing-existing.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+{{<Aside type="note">}}
+
+Cloudflare Calls only bills for traffic from Cloudflare's servers to your client, called `egressBytes`.
+
+{{</Aside>}}
+
+You should write a single query summing usage _per customer_ per time period and returning a single value. Do not use queries listing usage for more than a single customer such as aggregate queries that cover more than one customer.
+
+Here's an example of an _incorrect_ query that gets the usage for 10 customers. This is incorrect because the usage of one customer could affect the sampling of another customer. Results in the whole query will be statistically significant, however if there is a user with outside usage in the query this may cause a lower sample rate than required to be applied.
```
Comment (nils-ohlmeier) [inline content/calls/turn/replacing-existing.md]: The query sample below looks like it collecting data for 100 customers, not just 10.

**Review by nils-ohlmeier — APPROVED** [review]
You should probably fix the 10 vs 100 thing. But otherwise looks good.

File: content/calls/turn/replacing-existing.md (hunk 2 above)
Comment (renandincer) [reply content/calls/turn/replacing-existing.md]: also outside -> outsize. Maybe I can get AI to rephrase this whole area. Even I found it hard to understand :)

File: content/calls/turn/what-is-turn.md (hunk 1 above)
Comment (renandincer) [reply content/calls/turn/what-is-turn.md:26]: Not sure. I am not sure if it's too important.

Comment (renandincer) [reply content/calls/turn/what-is-turn.md:26]: I just google'd around and did not found a good document that explains what turn is, that is not mega marketing

**Pull request merged**